lib/jsonrpc2/types.rb

```diff
@@ -125,8 +128,11 @@ def valid_result?(interface, method, value)
       if about[:returns].nil?
         return value.nil?
       end
```

Comment:
Nit: As it's surrounded by one-line conditionals, it would be pleasing to adjust this one too. 

```suggestion
      return value.nil? if about[:returns].nil?
```